apps/wallet/src/ui/app/pages/initialize/backup/index.tsx

```diff
@@ -54,55 +53,127 @@ const BackupPage = ({ mode = 'created' }: BackupPageProps) => {
     }, [dispatch, mode, guardsLoading]);
     return (
         <Loading loading={guardsLoading}>
-            <CardLayout
-                icon="success"
-                title={`Wallet ${
-                    mode === 'imported' ? 'Imported' : 'Created'
-                } Successfully!`}
-                subtitle={mode === 'created' ? 'Recovery Phrase' : undefined}
-            >
-                {mode === 'created' ? (
-                    <>
-                        <Loading loading={loading}>
-                            {mnemonic ? (
-                                <div className={st.mnemonic}>
-                                    {mnemonic}
-                                    <CopyToClipboard
-                                        txt={mnemonic}
-                                        className={st.copy}
-                                        mode="plain"
-                                    >
-                                        COPY
-                                    </CopyToClipboard>
-                                </div>
-                            ) : (
-                                <Alert>{error}</Alert>
-                            )}
-                        </Loading>
-                        <div className={st.info}>
-                            Your recovery phrase makes it easy to back up and
-                            restore your account.
-                        </div>
-                        <div className={st.info}>
-                            <div className={st.infoCaption}>WARNING</div>
-                            Never disclose your secret recovery phrase. Anyone
-                            with the passphrase can take over your account
-                            forever.
+            <div className="flex flex-col flex-nowrap items-center justify-center">
+                <div className="p-7.5 pt-10 rounded-20 bg-alice-blue shadow-wallet-content flex flex-col flex-nowrap items-center justify-center w-popup-width h-popup-height">
```

Comment:
This set of classes seems like it's used a few times, could it be extracted into a component?

Reply:
I plan to update and combine with the existing `CardLayout` component once I update that component to account for different styles

Reply:
I think we can just adapt the Cardlayout for the UI changes/requirements, (probably bg colors and text styles?) And make sure that views can adapt using the available space to avoid issues like the following

https://user-images.githubusercontent.com/10210143/211605870-d6db77db-2150-4839-9f51-8cd05c3279e4.mov